src/components/drop/hoc/DropWithDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import {SlideY} from '../../../animations/SlideY';
+import {IReactVaporState} from '../../../ReactVapor';
+import {ConfigSupplier} from '../../../utils/HocUtils';
+import {PropsToOmitUtils} from '../../../utils/react/PropsToOmitUtils';
+import {ReduxConnect} from '../../../utils/ReduxUtils';
+import {IDropOwnProps, IDropProps, IDropStateProps} from '../Drop';
+import {DropPodPosition} from '../DropPod';
+import {DropSelectors} from '../redux/DropReducers';
+
+type TooltipWithDropdownComponent<T = any> = React.ComponentClass<IDropProps>;
+
+const mapStateToProps = (state: IReactVaporState, {id}: IDropOwnProps): IDropStateProps => ({
+    isOpen: DropSelectors.isOpen(state, {id}),
+});
+
+export const dropFormDropdown = (supplier: ConfigSupplier = {}) => (Component: TooltipWithDropdownComponent): TooltipWithDropdownComponent => {

```

Comment:
Not sure I like the name drop from dropdown, could we simply use `Dropdown` ?

Reply:
Or dropWithDropdown